code/integration-tests/local-integration-tests/src/cross_chain_transfer.rs

```diff
@@ -1072,17 +1065,165 @@ fn sibling_shib_to_transfer() {
 			WeightLimit::Unlimited,
 		));
 		assert_eq!(
-			<Tokens as MultiInspect<_>>::balance(sibling_asset_id, &AccountId::from(bob())),
+			<Tokens as FungiblesInspect<_>>::balance(sibling_asset_id, &AccountId::from(bob())),
 			total_issuance - transfer_amount
 		);
 	});
 
 	This::execute_with(|| {
 		use this_runtime::*;
 		log::info!(target: "bdd", "Then Bob gets some SHIB on Dali");
-		let fee = this_runtime::xcmp::xcm_asset_fee_estimator(5, remote_sibling_asset_id);
+		let fee = xcmp::xcm_asset_fee_estimator(5, remote_sibling_asset_id);
+		assert_gt!(transfer_amount, fee);
+		let balance = Tokens::free_balance(remote_sibling_asset_id, &AccountId::from(bob()));
+		assert_lt_by!(balance, transfer_amount, fee);
+	});
+}
+
+#[test]
+fn transfer_unknown_token_from_known_origin_ends_up_in_unknown_tokens() {
+	simtest();
+	let total_issuance = 3_500_000_000_000_000;
+	let transfer_amount = SHIB::ONE;
+	let sibling_asset_id = Sibling::execute_with(|| {
+		use sibling_runtime::*;
+		log::info!(target: "bdd", "Given one well-known/registered/sufficient/payable Sibling chain asset");
+		let sibling_asset_id =
+			CurrencyFactory::create(RangeId::TOKENS).expect("Valid range and ED; QED");
```

Comment:
NIT
Currency Factory no longer takes ED. This and other `expect` calls should reflect that.

Reply:
thanks. will fix in all places the text next prs